### ISSUE TITLE
reproducibility - guidance  - advice

### ISSUE THREAD

**Issue by clives** [NLeSC/mcfly#190]
Hi, first, thanks for this great library. 
I have only one doubt, the **reproducibility**,
I didn't see any recommandation in the documentation.
I tried to see the random seed, the numpy random seed,.. the result
are still way different for the same random seed. 
not sure if that's an issue or if there is a missing configuration on my side?

test executed: time series classification - raising or decraising - simple exampe

Thanks

**Comment by cwmeijer** [comment]
Hi,
I often ran the tutorial, as a test and at work shops and got exactly the same models on several different machines. Untrained models that is. You have to be sure to do the exact same calls in the same order right after setting the seed. Have a look at the tutorial to see how we set the seed there (http://localhost:8888/notebooks/mcfly-tutorial/notebooks/tutorial/tutorial.ipynb).
I don't think the training process will be the identical. I think Tensorflow uses a different source for random numbers. I haven't tested this though. So indeed the trained models will differ but their hyperparameters (chosen by mcfly through numpy) will be the same.
Hope this helps.
